# DEF-D Fysieke Ontwerpopdracht 2 invul template

## Groep informatie

Aan het einde van deze ontwerpopdracht lever je als groep één gezamenlijk notebook in (als in *PR* op GitHub en de link op Brightspace). Zet jullie groepsnummer, studienummers en namen in onderstaande tabel zodat wij zeker weten dat het groepsvormen goed is gegaan.

| Groep: 13     |  |
| :-------------|:-------------|
| Isha Schneiders| 6384838 |
| Laura Otto| 6488463 |
| Max van Spaendonck| 6579353 |
| Planning Groep: 13     |Tijdstip / Tijdspanne  |
|---|---|
| sanity check 1 | 11.15- |
| sanity check 2 | 11.30- |
| sanity check 3 | 12.15 |
| ontwerpen|13.45-14.45|
| bouwen en meten |14.45-16.30|
| opruimen/tijd over | 16.30-17.30 |
| Pauze 1| 12.30-13.30 |


## *Opdracht 1*: Sanity check 3

#### Voeg hier een foto van de schets van je plaatcondensator in. Geef daarbij ook duidelijk de waarde van $A$ en $d$ aan.
<img src="schets_plaat.jpg" alt="schets student 1" width="400"/>

In [1]:
# Schrijf hier een python functie die de capaciteit van een plaatcondensator uitrekent. Zorg dat de functie
# als input de ontwerpparameters van de vlakke plaat formule accepteert (oppervlakte, afstand en diëlektrische
# constante) en als output de capaciteit van de vlakke plaat condensator met die ontwerpparameters.
# Je gaat deze formule de rest van de dag gebruiken: controleer hem goed!
import numpy as np

def vlakkeplaat(epsilon,L,B,d):
    '''
    epsilon = permitiviteit medium
    L = lengte in m
    B = breedte in m
    d = afstand tussen platen in m
    '''
    C = epsilon*B*L/d 
    return C

def V_out(C,V_in_amplitude,freq,R):
    return C*V_in_amplitude*2*np.pi*freq*R
# Optioneel: schrijf je functie zodanig dat je arrays als input kan geven en (dus) ook arrays als output.

def C_gemeten(V_out,freq,R,V_in):
    return V_out/(2*np.pi*freq*R*V_in)

In [2]:
# Gebruik je formule om uit te rekenen wat je verwacht dat de capaciteit van jullie vlakke plaat condensatoren
# gaat zijn.
epsilon_0=8.854e-12 #F/m 
epsilon_R=3 #voor papier is het 3 keer zo veel als voor vacuum   
epsilon_papier=epsilon_R*epsilon_0
L= 10e-2 #m
B= 10e-2 #m
d= 1e-3 #m
#C = [functieNaam(2.0), functieNaam(4.7),functieNaam(1.2)]
C_v=vlakkeplaat(epsilon_papier,L,B,d)
print(f"Berekende capaciteit plaatcondensator groep 13 is {C_v:.1e} F")

V_out_amp=V_out(C_v,8,100e3,1000)
print("berkenende V_out_amplitude is %.2f V"%V_out_amp)

C_out=C_gemeten(0.7,100e3,1000,8)
print("gemeten capaciteit van de ocndensator is %e F"%C_out)

Berekende capaciteit plaatcondensator groep 13 is 2.7e-10 F
berkenende V_out_amplitude is 1.34 V
gemeten capaciteit van de ocndensator is 1.392606e-10 F


#### Voeg hier een foto van je plaatcondensator en van de meting op de scope in. Dus 2 foto's.
![scope meting 1](meting_plaat_condensator.jpeg)
![scope meting 2](opsteling_plaat_condensator.jpeg)

### Meting capaciteit
Vul onderstaande tabel in. Bij Sanity check 2 heb je afgeleid hoe je van berekende capaciteit naar berekend voltage gaat en terug. 


| Namen         | Berekende capaciteit   | Berekend Voltage(amplitude) | Gemeten Voltage(amplitude) | Gemeten capaciteit   |
|-------------|-------------:|-------------:|-------------:|-------------:|
| groep 13| 270 pF | 1.34  V| 0.7 V |139 pF| 

## *Opdracht 2:* Synthese.

#### Voeg hier voor een foto van je schets(en) van minimaal drie sensoren in.
![drie schetsen](tudelftLogo.png)


## Opdracht 3: Welke sensor gaan jullie maken?

Pas deze tekst aan: Wij gaan een sensor maken die X meet.

## Opdracht 4: Uitgewerkte schets (afmetingen!) en plot van de ontwerpgrafiek.

#### Voeg hier een foto van jullie uitgewerkte  schets van de sensor die jullie gaan maken in
![uitgewerkte schets](tudelftLogo.png)

In [3]:
# Plot hier de verwachte ontwerpgrafiek van jullie sensoren.

## Opdracht 5: Invullen tabel LTSpice.

|         | Capaciteit waarde (pF)  | LTSpice berekend output Vpp (mV)|
|-------------|-------------:|-------------:|
| minmale capaciteit| ? |? |
| gemiddelde capaciteit| ? |? |
| maximale capaciteit| ? |? |

## Opdracht 6: Foto's werkende opstelling.

#### Voeg hier de foto's van je sensoren in actie in.
![meting 1](tudelftLogo.png)
![meting 2](tudelftLogo.png)

## Opdracht 7: Plot de kalibratiegrafiek.

In [4]:
# Plot hier je gemeten kalibratiegrafiek en plot in dezelfde figuur ook je verwachte ontwerpgrafiek.

## Opdracht 8: Vul de tekst aan.

Onze sensor kan nog verbetered worden door de volgende verbeteringen (minimaal 2):

* verbetering 1
* verbetering 2

## Opdracht 9: Uiteindelijke ontwerp en kalibratiegrafiek.

#### Voeg hier een foto van je uiteindelijke sensor in.
![foto sensor](tudelftLogo.png)

In [5]:
# Plot hier je oorspronkelijke en je uiteindelijke kalibratiegrafieken, fit daar een bijpassende lijn door
# en zet ook de verwachte ontwerpgrafiek erbij

## Opdracht 10: Vul hieronder de conclusie in.

Vul hier de conclusie in

## Opdracht 11: Kopieer hieronder de leerdoelen.
1. Het kunnen gebruiken van de eigenschappen van een elektrische component in een ontwerpproces om een fysische grootheid te meten.
2. Kennis hebben van gedrag van elektrische componenten en meetopstellingen.
3. Onderdelen van een datasheet zoals een kalibratiegrafiek opstellen, gevoeligheid en onzekerheid beschrijven.

kopieer hier de cel(len) waaruit blijkt dat jullie leerdoel 1 beheersen.

kopieer hier de cel(len) waaruit blijkt dat jullie leerdoel 2 beheersen.

kopieer hier de cel(len) waaruit blijkt dat jullie leerdoel 3 beheersen.